frontend/src/routes/Infrastructure/Clusters/ClusterPools/ClusterPools.tsx

```diff
@@ -250,6 +256,26 @@ export function ClusterPoolsTable(props: {
                                     },
                                 ],
                             },
+                            {
+                                cells: [
+                                    {
+                                        title: (
+                                            <>
+                                                {clusterPoolClaims.length > 0 && (
+                                                    <div style={{ marginTop: '16px', marginBottom: '16px' }}>
+                                                        <TextContent>
+                                                            <Text component={TextVariants.h3}>
+                                                                {t('clusterClaims')}
```

Comment:
This heading needs to be **Pending claims** so it is clear it is not a list of all claims